src/PTFAnalysis.cpp

```diff
@@ -4,20 +4,37 @@
 #include "Utilities.hpp"
 #include "TVirtualFFT.h"
 #include "PulseFinding.hpp"
+#include "TCanvas.h"
```

Comment:
There were a couple include files added TCanvas, TH1D, TStyle which I think can be removed